flink-libraries/flink-table/src/main/scala/org/apache/flink/table/functions/AggregateFunction.scala

```diff
@@ -35,6 +36,18 @@ abstract class AggregateFunction[T] extends UserDefinedFunction {
   def createAccumulator(): Accumulator
 
   /**
+    * Retract the input values from the accumulator instance.
```

Comment:
Do we want to mention that only previously `accumulated` values need to be `retracted`?
Users should not check for that but can use this assumption to implement more efficient functions.